docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -75,6 +76,13 @@ On macOS, point the variable to the `Home` sub-directory:
 ----
 export GRAALVM_HOME=$HOME/Development/graalvm/Contents/Home/
 ----
++
+On Windows, you'll have to go through the Control Panel to set your environment variables.
++
+[TIP]
+====
+Installing via scoop will do this for you.
```

Comment:
a google search didn't reveal to me how to install graalvm with scoop - got a pointer ?

Reply:
Above I linked to the scoop bucket for Java. Once you install scoop, you'll have to enable the bucket `scoop bucket add java` and then install graal: `scoop install graalvm`. That installs the latest release (jdk 11), but theres also `graalvm-nightly`, `graalvm8`, `graalvm8-nightly`, `graalvm11`, and `graalvm11-nightly`.

Reply:
very nice! good to know!